docs/guides/api-graphql/subscriptions-by-id.md

```diff
@@ -64,21 +64,6 @@ type Subscription {
 
 ```
 
-Now you can subscribe to the comments for a specific post using the post id:
-
-```js
-import { API } from 'aws-amplify';
-import { onCommentByPostId } from './graphql/subscriptions';
-
-API.graphql({
-  query: onCommentByPostId,
-  variables: {
-    postCommentsId: "12345"
-  }
-})
-.subscribe({
-  next: data => {
-    console.log('data: ', data)
-  }
-})
-```
\ No newline at end of file
+<inline-fragment platform="ios" src="~/guides/api-graphql/fragments/ios/subscriptions-by-id.md"></inline-fragment>
+<inline-fragment platform="js" src="~/guides/api-graphql/fragments/js/subscriptions-by-id.md"></inline-fragment>
+<inline-fragment platform="android" src="~/guides/api-graphql/fragments/android/subscriptions-by-id.md"></inline-fragment>
```

Comment:
Currently, the Android documentation is in Swift. How should we proceed?
1. Include it anyway;
2. Update this PR with examples in Java and Kotlin;
3. Exclude line 69 and do add the Android examples later.

Is (3) possible? What happens if you don't include an Android fragment right now?

Reply:
Hi @jamesonwilliams! I would prefer 2. and have already asked @richardmcclellan to review the PR and provide the Java/Kotlin code snippets. This way we can improve the existing docs based on my first iteration and then get the code in place

Reply:
I've opened a PR here! https://github.com/aws-amplify/docs/pull/2538

Reply:
Thanks @richardmcclellan!